### ISSUE TITLE
Double Speed for forcing fp32 with Zluda on flux

### ISSUE THREAD

**Issue by DeeH0pe** [lshqqytiger/stable-diffusion-webui-amdgpu-forge#27]
Anyone else noticed a speed boost while consuming more vram with this commandline args?

--use-zluda --attention-quad --all-in-fp32

Im about around 3s/it faster than with bfloat16

![grafik](https://github.com/user-attachments/assets/a71b1132-3b10-4a06-875f-4cffd6b16828)

**Comment by mirh** [comment]
Yes, you aren't the first one https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1684

**Comment by VeteranXT** [comment]
You can use Q4? How?

**Comment by DeeH0pe** [comment]
> You can use Q4? How?

Just put the Q4 in Stable-diffusion folder. Open launch.py and add these 2 lines and adjust your Path to the packages_3rdparty folder:

import sys
sys.path.append(r"Your_Path\packages_3rdparty")

**Comment by bigwinboy** [comment]
> > You can use Q4? How?
> 
> Just put the Q4 in Stable-diffusion folder. Open launch.py and add these 2 lines and adjust your Path to the packages_3rdparty folder:
> 
> import sys sys.path.append(r"Your_Path\packages_3rdparty")

Hello, How is the sampler etc set up please, I am able to run it but all that comes out is a black picture.

**Comment by mirh** [comment]
Black picture may be https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1278

**Comment by bigwinboy** [comment]
> Black picture may be [lllyasviel#1278](https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1278)

Thank you, successfully generated images after adding the command --all-in-fp32 to COMMANDLINE_ARGS= in webui-user.bat.
